src/hb-ot-metrics.cc

```diff
@@ -58,53 +57,48 @@ hb_ot_metrics_get (hb_font_t       *font,
   switch (metrics_tag)
   {
 #define GET_METRICS(TABLE, ATTR, DIR) \
```

Comment:
Might as well define two directional macros that call this one. `GET_X_METRIC` / `GET_Y_METRIC`.  And yes, I'm not happy re 'metrics' vs 'metric' naming.  Not sure what I want.

Reply:
Done